src/jose/algorithms/ed25519.ts

```diff
@@ -1,22 +1,7 @@
 import * as ed25519 from '@noble/ed25519';
 import { base64url } from 'multiformats/bases/base64';
 
-/**
- * An Ed25519 public key in JWK format.
- */
-export type JwkEd25519Public = {
-  kty: 'OKP';
-  crv: 'Ed25519';
-  x: string;
-};
-
-/**
- * An Ed25519 private key in JWK format.
- */
-export type JwkEd25519Private = JwkEd25519Public & {
-  d: string; // Only used by a private key
-};
-
+import type { JwkEd25519Private, JwkEd25519Public, Signfn, VerifyFn  } from '../types';
```

Comment:
I guess the intent was to move types into the same place? But feels strange here that `JwkEd25519Private` and `JwkEd25519Public` need to be imported from a place that shouldn't have alg specific knowledge.

Reply:
true. i feel the same way. think it makes more sense to put these in `algorithms/types.ts`?

Reply:
Same as the other question on `types.ts` why can't `ed25519.ts`  directly export ed25519 related types?

Reply:
went a slightly different route but it did:
* address the weirdness
* reduce the number of files that need to be touched to add a new algo down to 2 (though i'm not sure how often we'll be adding keys)
* prepared the jose dir for encryption-related algos


you may like it less but i think it should work for now 🤣 https://github.com/TBD54566975/dwn-sdk-js/commit/2456261f3856e561e5af7b67190dd02a598faa33